### ISSUE TITLE
Autocomplete scss variables

### ISSUE THREAD

**Issue by napalmpapalam** [mrmlnc/vscode-scss#121]
- VS Code Version: 1.46.1
- SCSS IntelliSense Version: 0.9.0
- Operating System: Linux

When I start to type scss variable that name starts on $ symbol, and trigger autocomplete on Tab key, It autocomplete with 2 dollar symbol in start of the name, like this: `color: $$col-input-field`

**Comment by mattaningram** [comment]
Yeah a few of us are having this problem: https://github.com/mrmlnc/vscode-scss/issues/118

**Comment by anywayscode** [comment]
Same problem, I really hate it, that stop write fastly

**Comment by YongzeYao** [comment]
Same here. It is really annoying.

**Comment by napalmpapalam** [comment]
any news?

**Comment by marblewraith** [comment]
VS Code Version: 1.56.0
SCSS IntelliSense Version: 0.9.1
Operating System: Windows 10

I found something weird, rather than trying to explain from scratch i made a gif (recommend expanding to full screen):

![6fiU1ESNbJ](https://user-images.githubusercontent.com/45786628/117380439-0653ce00-af1d-11eb-8239-211592196228.gif)

Notice the 2 intellisense entries for `$root-font`.

The one with the all the `../../../..` behaves correctly i.e. inserts the variable with no extra `$` symbol.

The one that shows reference to the file itself (`funkymix/_defaults.scss`), that one bugs out with the extra `$` symbol.

Also while we're at it, i question why it displays 2 entries for the same variable at all? It should only show 1, as there's only 1 variable, right?

I hope this can be of help debugging 😊

**Comment by Omcsesz** [comment]
Is there any improvement on this issue?

**Comment by thorstensson** [comment]
I am actually having this issue now, in 2025 with intellisense (latest) in latest VS code in an SCSS file.

Anyone else seeing this still; tried lots of settings.

.T
